pkg/chains/signing/iface.go

```diff
@@ -18,6 +18,7 @@ import "github.com/sigstore/sigstore/pkg/signature"
 type Signer interface {
 	signature.Signer
 	Type() string
+	Cert() string
```

Comment:
nit: should we future proof this a bit by making it cert and chain?

Reply:
yee! done